app/lib/rocketchat.js

```diff
@@ -90,7 +90,22 @@ const RocketChat = {
 	},
 	async getServerInfo(server) {
 		try {
-			const result = await fetch(`${ server }/api/info`).then(response => response.json());
+			let result = await fetch(`${ server }/api/info`)
+				.then(response => response.text());
+			try {
+				result = JSON.parse(result);
+				if (!{}.hasOwnProperty.call(result, 'success')) {
```

Comment:
I'm lazy and I think this is too complicated.
What do you think of `!(result && result.success)`?